### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**
[DEMO LINK](https://igor-stasiv.github.io/react_list-of-goods/)

### CODE REVIEW

**Review by pkuzmenkoo — APPROVED** [review]
good decision, everything is clear

File: src/App.js [hunk 1]
```diff
@@ -14,11 +14,129 @@ const goodsFromServer = [
...
+            Set filter by word length:
+            <select
+              value={lengthFilter}
+              onChange={({ target }) => {
+                this.setState({
+                  lengthFilter: target.value,
+                });
+              }}
+            >
+              <option>1</option>
```
Comment (varseniuk) [inline src/App.js:96]: Let's not write so many lines of code, but use an array of numbers from 1 to 10 and map `options` from it

**Review by varseniuk — CHANGES_REQUESTED** [review]
Good code! 👍 
But let's separate the logic to different components (read my comment)

File: src/components/GoodsList.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const numbers = [];
+
+for (let i = 1; i <= 10; i += 1) {
+  numbers.push(i);
```
Comment (varseniuk) [inline src/components/GoodsList.jsx:7]: You could use `new Array(10)` with `fill` method as a variant

**Review by varseniuk — APPROVED** [review]
Good job!

**Pull request closed without merging**